config_defaults_inc.php

```diff
@@ -1214,6 +1214,16 @@
  */
 $g_private_news_threshold = DEVELOPER;
 
+############
+# Markdown #
+############
+
+/**
+ * Enables markdown support - once enabled all issue titles, text, and notes are formatted based on
+ * markdown syntax.  Default to OFF since it is experimental feature for now.
+ */
+$g_markdown_enabled = OFF;
```

Comment:
@jllano After moving markdown to a plugin do we still need the $g_markdown_enabled config?  I was expecting it will be dropped in favor of installing / uninstalling the plugin.